code/src/hooks/use-input-visibility/useInputVisibility.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+
+import { InputAdornment } from "@mui/material";
+import VisibilityIcon from "@mui/icons-material/Visibility";
+import VisibilityOffIcon from "@mui/icons-material/VisibilityOff";
+
+import AppIconButton from "@/components/app-icon-button/AppIconButton";
+
+const useInputVisibility = () => {
+  const [showInputText, setShowInputText] = useState(false);
```

Comment:
I recommend using `shouldShowInputText` because at first glance `showInputText` looks like function, not boolean

Reply:
Fixed!